linters/elixir/.credo.exs

```diff
@@ -0,0 +1,21 @@
+%{
+  configs: [
+    %{
+      name: "default",
+      files: %{
+        included: ["lib/", "src/", "web/", "apps/"],
+        excluded: [~r"/_build/", ~r"/deps/"]
+      },
+      requires: [],
+      check_for_updates: true,
+      strict: true,
+      color: true,
+
+      checks: [
+        {Credo.Check.Design.TagTODO, exit_status: 0},
```

Comment:
Well, someone said that a TODO is a fuck you to the other developers. That being said, I myself have added them from time to time as a reminder to comeback to it.

I don't have a strong opinion either way, but do think that breaking the build because of it is a tad too much for my liking. So in principle I'm ok with this.